### PR TITLE
Add 3 custom attributes to ccpDialog 

### PR DESCRIPTION
**Pull request by nqnhat1988**
I made this PR for

1. Set custom string for title, search hint, no result message

### CODE REVIEW

**Comment by hbb20** [comment]
@nqnhat1988, thank you so much for your valuable contribution. 👍  We will review and implement the feature in the upcoming release.

File: app/src/main/res/layout/fragment_introduction.xml [hunk 1]
```diff
@@ -114,7 +114,6 @@
                                 android:id="@+id/ccp"
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
-                                app:ccpDialog_titleString="Hey you"
```
Comment (hbb20) [inline app/src/main/res/layout/fragment_introduction.xml]: Nice 🤣

**Comment by nqnhat1988** [comment]
> @nqnhat1988, thank you so much for your valuable contribution. 👍 We will review and implement the feature in the upcoming release.

@hbb20 Could I know when the next release date? Because I also need to release my app by the end of this month and I just don't want to import `ccp` as a project into the codebase :)

**Comment by hbb20** [comment]
Sure, we will release a new version before the last week of the month. At least some good work should be done during school Spring Break, right? 😉 

One more thing, we might modify your this implementation. As this library is used for many languages, we would like provide a way to support custom string for multiple languages as well. So you might need to modify your code after the release of the library. But that will be an easy change.

**Comment by hbb20** [comment]
@nqnhat1988, we just released new version 2.2.5 of the library. It allows developers to provide custom values for these 3 properties. 

Though, implementation is different from the original PR due to multi-language support. Please go through the [read me](https://github.com/hbb20/CountryCodePickerProject/wiki/Custom-Dialog-Title--%7C-Search-Hint-%7C-Empty-result-ACK). Please let me know if it will work for you. Thanks.

**Pull request merged**